examples/v1beta1/pipelineruns/pipelinerun-with-extra-params.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: pipeline-with-extra-params
+spec:
+  params:
+  - name: pl-param-x
+    type: string
+  - name: pl-param-y
+    type: string
+  tasks:
+  - name: add-params
+    taskRef:
+      name: add-params
+    params:
+    - name: a
+      value: "$(params.pl-param-x)"
+    - name: b
+      value: "$(params.pl-param-y)"
+---
+apiVersion: tekton.dev/v1alpha1
```

Comment:
please change the apiVersion to `v1beta1` here

Reply:
ah I missed that :sweat_smile: